specification/registry/xr.xml

```diff
@@ -3516,6 +3537,7 @@ maintained in the default branch of the Khronos OpenXR GitHub project.
             <param><type>XrSpatialAnchorStoreConnectionMSFT</type> <name>spatialAnchorStore</name></param>
         </command>
 
+
```

Comment:
you can probably drop this extra line.

Reply:
Extra lines removed.
Regarding "reflowing" I'd let you do it to avoid my mistakes (I'm barely familiar with such term).